docs/Unit-tests.md

```diff
@@ -0,0 +1,30 @@
+# Unit tests guide
+
+
+## Prerequisites
+
+You must have [Rake](http://rake.rubyforge.org/) installed on your testing environment before launch unit tests, you can install it using gem.
+
+You can see all possible commands using `Rake -T`.
+
+Before launch unit tests, you have to set 3 env vars :
+
+* `TEST_MOUNTPOINT`: the svfs mountpoint (without the slash at the end).
+* `TEST_SEG_SIZE`: segmented file size value (in megabytes).
+* `TEST_NSEG_SIZE`: standard file size value.
+
+Now the test container will be created automatically.
+
+## Launch unit tests
+
+For launch unit tests, you just have to use `rake test` command after have follow the prerequisites instructions.
```

Comment:
For launch unit tests => In order to run unit tests
after have follow the prerequisites instructions => once you have followed previous instructions.